### DISCUSSION TITLE
希望增加google浏览器加速

### DISCUSSION THREAD

**Discussion by ZHENYU-TANG** [BeyondDimension/SteamTools#1441] [General]
希望增加google浏览器加速，更方便。

**Comment by giegie12** [comment]
你想的是对谷歌的加速吧，这个不行，除非开发者想喝茶

**Comment by lucien-gh** [comment]
最希望能把Google翻译加速一下，把Google翻译网页版和api都加速

**Reply by monki912** [reply]
加速谷歌作者得进局子

**Comment by elarbor** [comment]
谷歌翻译可以

![image](https://github.com/user-attachments/assets/3de37449-ff74-440d-8e80-3feb71d15e23)

**Comment by TravelTibet** [comment]
- 谷歌不可能✋🏿
